### ISSUE TITLE
About height of drone

### ISSUE THREAD

**Issue by darkangel20156** [dji-sdk/Mobile-SDK-Android-V5#303]
When I use KeyAircraftLocation3D to get height of drone, but the result is not true. Specifically, when the real height of drone is 1m, the result is 0.0, and sometimes it is -0.1; -0.2m.
![image](https://github.com/dji-sdk/Mobile-SDK-Android-V5/assets/123035738/bb923fab-de4f-45fe-b567-9b81820547b4)

So I want to know how the height is calculated?

**Comment by dji-lyt** [comment]
What is the actual height you are referring to? When the aircraft reaches a height of 1 meter after takeoff, does KeyAircraftLocation3D read a height of 0?

Could you please provide us with the likelihood of this issue occurring? And could you also share a short video to confirm the issue's behavior? I would recommend using the example code from MSDK V5, where you can view aircraft diagnostic information and height on the default delay page.

The height from KeyAircraftLocation3D is relative to the takeoff point.

![image](https://github.com/dji-sdk/Mobile-SDK-Android-V5/assets/104296295/3e294f65-513a-4ba5-b434-d42d00254725)

**Comment by darkangel20156** [comment]
I tested in 7th floor of a building (it is in door, and low GPS, I got GPS level and the result is level_0). When I take off the drone (dji mini 3), if according to the documents, the take-off height is (1.2m) but when I get the altitude value on the screen, the value is not 1.2.
In addition to keyLocation3D, I also used KeyAltitude (MSDK 5.6) and the result did not give the exact height of 1.2m.
